site/using-weave/finding-adding-hosts-dynamically.md

```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
 menu_order: 90
 ---
 
-
-To add a host to an existing Weave network, simply launch 
-Weave Net on the host, and then supply the address of at least 
+To add a host to an existing Weave network, launch 
+Weave Net on the host, and supply the address of at least 
 one host. Weave Net automatically discovers any other hosts in 
 the network and establishes connections with them if it 
 can (in order to avoid unnecessary multi-hop routing).
 
-In some situations all existing Weave Net hosts may be 
+In some situations existing Weave Net hosts may be 
 unreachable from the new host due to firewalls, etc. 
 However, it is still possible to add the new host, 
 provided that inverse connections, for example, 
 from existing hosts to the new hosts, are available. 
 
 To accomplish this, launch Weave Net onto the new host 
-without supplying any additional addresses.  And then, from one 
+without supplying any additional addresses and then, from one 
 of the existing hosts run:
 
     host# weave connect $NEW_HOST
 
-Other hosts on the Weave network will automatically attempt
+Any other existing hosts on the Weave network will attempt
 to establish connections to the new host as well. 
 
-Alternatively, you can also instruct a peer to forget a 
+###Instructing Peers to Forget a Host
```

Comment:
It is odd to have this heading but no 'Instructing Peers to Connect to a Host' before that.